docs/user/alerting/alert-types.asciidoc

```diff
@@ -1,159 +1,28 @@
 [role="xpack"]
 [[alert-types]]
-== Standard stack alert types
+== Alert types
 
-{kib} supplies alert types in two ways: some are built into {kib} (these are known as stack alerts), while domain-specific alert types are registered by {kib} apps such as <<xpack-apm,*APM*>>, <<xpack-ml,*{ml-cap}*>>, <<metrics-app,*Metrics*>>, and <<uptime-app,*Uptime*>>.
+{kib} supplies alert types in two ways: some are built into {kib} (these are known as stack alerts), while domain-specific alert types are registered by {kib} apps.
```

Comment:
I'd format this as follows"

Kibana provides two types of alerts:

- Stack alerts, which are built into {kib}
- Domain-specific alerts, which are registered by {kib} apps.